File: frontend/src/app/terapija/TerapijaGetAllResponse.ts
export interface TerapijaGetAllResponse
{
 terapije:TerapijaGetAllResponseTerapija[]
}

export interface TerapijaGetAllResponseTerapija
{
    terapijaId: number;
    opis: string;
    doktorId: number;
    lijekId: number | null;
    korisnikDomaID:number;
    nacinPrimjene: string;
    vremenskiInterval: string;
}

File: DomZaStaraLicaApi/Endpoints/Terapija/GetAll/TerapijaGetAllResponse.cs
using DomZaStaraLicaApi.Data.Models;
using System.ComponentModel.DataAnnotations.Schema;

namespace DomZaStaraLicaApi.Endpoints.Terapija.GetAll
{
    public class TerapijaGetAllResponse
    {
       public List<TerapijaGetAllResponseTerapija> Terapije {  get; set; }
    }

    public class TerapijaGetAllResponseTerapija
    {
        public int TerapijaId { get; set; }
        public string Opis { get; set; }
        public int DoktorId { get; set; }
        public int KorisnikDomaID { get; set; }
        public string NacinPrimjene { get; set; }
        public string VremenskiInterval { get; set; }


    }
}
